encrypted-media-respec.html

```diff
@@ -3053,13 +3067,11 @@
               </ol>
             </li>
             <li>
-              <p>Run the following steps:</p>
-              <p class="note">These steps are reached when there is no key that is <a def-id="usable-for-decryption"></a> for <var>block</var>.</p>
-              <ol>
-                <li>
-                  <p>Run the <a def-id="wait-for-key-algorithm"></a> algorithm on the media element.</p>
-                </li>
-              </ol>
+              <p>Set the media element's <var>decryption requires usable key</var> value to <code>true</code>.</p>
+              <p class="note">This step is reached when there is no key that is <a def-id="usable-for-decryption"></a> for <var>block</var>.</p>
+              <div class="note">
+                <p>Once the user agent has rendered all the decrypted blocks that it can, it will run the <a def-id="wait-for-key-algorithm"></a> algorithm.</p>
```

Comment:
I think this should be "...rendered all blocks preceding the block which cannot be decrypted, ...", or similar, since there may be unencrypted blocks mixed in.